hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcCallContext.java

```diff
@@ -60,6 +61,13 @@ default Optional<String> getRequestUserName() {
     return getRequestUser().map(User::getShortName);
   }
 
+  /**
+   * Returns the TLS certificate(s) that the client presented to this HBase server when making its
+   * connection. TLS is orthogonal to Kerberos, so this is unrelated to
+   * {@link this#getRequestUser()}. Both, one, or neither may be present.
```

Comment:
@charlesconnell this is generating a javadoc warning. Can you fix before we merge?

Reply:
done